src/workspace-data/ImportStatus.ts

```diff
@@ -19,29 +19,45 @@ const ImportStatus = () => {
           key: job.jobId,
           job,
           onDone: () => {
-            asyncImportJobStore.update(_.reject({ jobId: job.jobId }));
+            asyncImportJobStore.update(_.reject({ jobId: job.jobId, targetWorkspace: job.targetWorkspace }));
```

Comment:
Could this be simplified to?

```suggestion
            asyncImportJobStore.update(_.reject(job));
```